images/ceph/Makefile

```diff
@@ -31,7 +31,7 @@ YQv3_VERSION = 3.4.1
 GOHOST := GOOS=$(GOHOSTOS) GOARCH=$(GOHOSTARCH) go
 MANIFESTS_DIR=../../deploy/examples
 
-TEMP := $(shell mktemp -d)
+BUILD_CONTEXT ?= $(shell mktemp -d)
```

Comment:
Doing this makes the temp dir change on each call. i think I need to eval it somehow